<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workspace</class>
 <widget class="QMainWindow" name="workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>971</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);

font: 87 9pt &quot;Raleway Black&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>GESTION DES SALLES</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_R">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convertir en Exel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cap">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure d'ouverture:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>437</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure de fermeture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Capacité de salle:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>440</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>Num salle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Capacité de salle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type d'objet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date paiment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>methode de paiment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>société</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Réference</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Liste_Materiel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>ID Salle:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Code QR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;Sitka Display&quot;;

</string>
      </property>
      <property name="text">
       <string>Listes des salles</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>437</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_S">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>500</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_S">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Arial&quot;;
</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_ho">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_hf">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="lineEdit_lm">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MODIFCATION</string>
     </attribute>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>ID Salle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Liste_Materiel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure d'ouverture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure de fermeture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Capacité de salle:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cap_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_hf_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_ho_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="lineEdit_lm_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>310</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
